Test.txt

```diff
@@ -1 +1,3 @@
-This is a test.
\ No newline at end of file
+This is how you change a file.
+This is how you would change the second line.
```

Comment:
Reply.

---
In reply to: [59603032](https://github.com/kivancmuslu/CFTest/pull/1#discussion_r59603032) [](ancestors = 59603032)